srml/support/procedural/src/storage/instance_trait.rs

```diff
@@ -28,6 +28,7 @@ pub(crate) const DEFAULT_INSTANTIABLE_TRAIT_NAME: &str = "__GeneratedInstantiabl
 // prefix for consts in trait Instance
 pub(crate) const PREFIX_FOR: &str = "PREFIX_FOR_";
 pub(crate) const HEAD_KEY_FOR: &str = "HEAD_KEY_FOR_";
+pub(crate) const LINKED_MAP_KEY_FORMAT_FOR: &str = "LINKED_MAP_KEY_FORMAT_FOR_";
```

Comment:
I don't think this should be in instance_trait, it is not related to instance trait it is just the name of a struct to generate along side storage struct.

The two other keywords are prefix for consts in the trait Instance.